packages/swapper/src/swappers/ButterSwap/endpoints.ts

```diff
@@ -0,0 +1,52 @@
+import { CHAIN_NAMESPACE, fromChainId } from '@shapeshiftoss/caip'
+import { evm } from '@shapeshiftoss/chain-adapters'
+import BigNumber from 'bignumber.js'
+
+import type { SwapperApi } from '../../types'
+import { getExecutableTradeStep, isExecutableTradeQuote } from '../../utils'
+import { checkTradeStatus } from './swapperApi/checkTradeStatus'
+import { getButterQuote } from './swapperApi/getTradeQuote'
+import { getTradeRate } from './swapperApi/getTradeRate'
+
+export const butterSwapApi: SwapperApi = {
+  getTradeQuote: getButterQuote,
+  getTradeRate,
```

Comment:
stylistic: no reason for one to be named `getButterQuote` and the other `getTradeRate`, could do `getTradeQuote` for the former for consistency

Reply:
https://github.com/shapeshift/web/pull/9799/#discussion_r2171319396 ?

Reply:
ser that's your PR link

Reply:
Fixed in https://github.com/shapeshift/web/commit/2722c339861c62c4fee1481c61f778c710020355